src/graph.ts

```diff
@@ -163,6 +167,12 @@ export class GraphBuilder {
                     `Something went wrong: resourceType ${resource.Type} does not equal CfnType ${cfnType}`,
                 );
             }
+            if (resource.Type === 'AWS::EC2::VPCCidrBlock') {
+                this.vpcCidrBlockNode = node;
+            }
+            if (resource.Type === 'AWS::EC2::VPC') {
+                this.vpcNode = node;
```

Comment:
Can a stack only have a single VPC?

Reply:
I think that case would be exceedingly rare, but I should be able to update this to handle that case